tests/cli/commands/test_task_command.py

```diff
@@ -184,6 +184,10 @@ def test_task_state(self):
             'tasks', 'state', 'example_bash_operator', 'runme_0',
             DEFAULT_DATE.isoformat()]))
 
+    def test_task_states_for_dag_run(self):
+        task_command.task_states_for_dag_run(self.parser.parse_args([
```

Comment:
Can you add some assertions?  I see that other tests do not have, but now we are trying to improve the CLI and I have already [made many changes](https://github.com/apache/airflow/pulls?q=is%3Apr+label%3Aarea%3ACLI+is%3Aclosed+author%3Amik-laj) that do it.
Here is sample of assertions in tests: https://github.com/apache/airflow/blob/master/tests/cli/commands/test_config_command.py#L38

Reply:
added